src/components/timeline/TimelineViewControls.svelte

```diff
@@ -106,6 +111,20 @@
   function onToggleDirectiveVisibility() {
     dispatch('toggleDirectiveVisibility', !allDirectivesVisible);
   }
+
+  function onCopyViewportURL() {
+    const targetUrl = new URL(window.location.href);
+    targetUrl.searchParams.set(SearchParameters.START_TIME, new Date(viewTimeRange.start).toISOString());
+    targetUrl.searchParams.set(SearchParameters.END_TIME, new Date(viewTimeRange.end).toISOString());
+    if ($selectedActivityDirective) {
```

Comment:
Should we also include selectedSpanId? Though we could leave that for the future since we don't technically support that yet (and still need to fix selected activity directive)?

Reply:
yes, that sounds good to me